docs/content/usage-reporting.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Usage Reporting
+description: |
+How to enable usage reporting for NGINX Ingress Controller and how to view the usage data through the API.weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: ["???"]
```

Comment:
```suggestion
```
The `docs:` frontmatter is being shuttered and replaced with something else when the new docs site is production ready.